aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-view.md

```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-view
 title: "Adding a View (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to change the Views and Layout pages and pass data from the Controller to the View."
```

Comment:
"Learn how to change the Views and Layout pages and pass data from the Controller to the View using Microsoft Visual Web Developer 2010 Express Service Pack 1."